utils/statediff/src/lib.rs

```diff
@@ -158,6 +160,133 @@ pub struct MinerState {
     pub deadline_cron_active: bool,
```

Comment:
Could we import those structures from `builtin-actors` instead? If some additional traits are needed, we could make a PR there.